evergreen.yml

```
@@ -208,6 +208,26 @@ functions:
              --validate-only --verbose ../../connector/ --log ../../logs 2>&1 | tee /dev/stdout \
              | grep "Validation succeeded." >/dev/null)
 
+  "update download center feed":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-tableau-connector
+        script: |
+          ${prepare_shell}
+          sed -i 's@{RELEASE_VERSION}@'${MONGO_TABLEAU_CONNECTOR_VERSION}'@' mongo-tableau-connector-downloads_template.json
+          echo "-------------------------"
+          cat mongo-tableau-connector-downloads_template.json
+          echo "-------------------------"
+    - command: s3.put
+      params:
+        aws_key: ${release_aws_key}
+        aws_secret: ${release_aws_secret}
+        local_file: mongo-tableau-connector/mongo-tableau-connector-downloads_template.json
+        remote_file: mongo-tableau-connector/mongo-tableau-connector-downloads.json
+        bucket: translators-connectors-releases
+        permissions: public-read
+        content_type: application/octet-stream
```

```suggestion
        content_type: application/json
```
I normally see `octet-stream` for our binaries.  This seems more appropriate, and [here's](https://github.com/mongodb/mongo-tools/blob/e62cf8866c5b98c45faf2fbe354703969aa076c6/common.yml#L411) one instance of json being used.

---

Just a comment here. `application/octet-stream` should be used for types that don't have a dedicated encoding according to [MDN(https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types). A full list of dedicated content-types can be found [here](https://www.iana.org/assignments/media-types/media-types.xhtml#application).